pkg/apis/pxc/v1/pxc_types.go

```diff
@@ -471,6 +471,7 @@ type PodSpec struct {
 	EnvVarsSecretName             string                        `json:"envVarsSecret,omitempty"`
 	TerminationGracePeriodSeconds *int64                        `json:"gracePeriod,omitempty"`
 	ForceUnsafeBootstrap          bool                          `json:"forceUnsafeBootstrap,omitempty"`
+    MySqlAllocator                string                        `json:"mySqlAllocator,omitempty"`
```

Comment:
I would change this field's name to `MySQLAllocator` and use `mysqlAllocator` in yaml

Reply:
also could you please run `go fmt` on this file?

Reply:
done